fused-effects.cabal

```diff
@@ -84,6 +84,17 @@ library
     Control.Effect.Throw
     Control.Effect.Trace
     Control.Effect.Writer
+  other-modules:
+    Control.Effect.Catch.Internal
+    Control.Effect.Choose.Internal
+    Control.Effect.Empty.Internal
+    Control.Effect.Error.Internal
+    Control.Effect.Lift.Internal
+    Control.Effect.NonDet.Internal
+    Control.Effect.Reader.Internal
+    Control.Effect.State.Internal
+    Control.Effect.Throw.Internal
+    Control.Effect.Writer.Internal
```

Comment:
We don’t export any of these modules, because they don’t define any functionality at all, only datatypes which are re-exported by the public interfaces.